Source/OxyPlot.Wpf/OxyPlot.Wpf.csproj

```diff
@@ -23,6 +23,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <DocumentationFile>bin\Debug\OxyPlot.Wpf.XML</DocumentationFile>
```

Comment:
if there is a reason to build this in the debug configuration, let us create a separate issue for it!  :-)